ScrollableTabBar.js

```diff
@@ -120,7 +126,7 @@ const ScrollableTabBar = React.createClass({
     const fontWeight = isTabActive ? 'bold' : 'normal';
 
     return <TouchableOpacity
-      key={name}
+      key={`${name}_${page}`}
```

Comment:
This is totally unrelated to the android-like behavior, but I need to be able to have two tabs with the same name. 

I should remove the change from here and make another PR for that.